mackerel-plugin-mssql/README.md

```diff
@@ -0,0 +1,18 @@
+mackerel-plugin-mssql
+=====================
+
+MSSQL custom metrics plugin for mackerel.io agent.
+
+## Synopsis
+
+```shell
+mackerel-plugin-mssql [-prefix=<PREFIX>] [-instance=<SQLSERVER|SQLEXPRESS>] [-tempfile=<tempfile>]
+```
+
```

Comment:
[q]
Is there any requirements for MSSQL versions/editions?
If any, it's better to be documented here.